docs/subvolume.md

```diff
@@ -0,0 +1,53 @@
+# Subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which have no parent-pvc attached to them.
+The command would list out all such subvolumes which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the subvolumes
+    * `--type=stale`: lists all the stale volume that can be deleted
+* `delete <subvolumes> <filesystem> <subvolumegroup>`:
+    [delete](#delete) stale subvolumes after user's confirmation.
```

Comment:
Is there a confirmation? I thought we just automatically delete it after checking that it's stale

Reply:
confirmation as in, we allow the user to pass the details, so that we don't delete it directly.